lib/api/utils/url.js

```diff
@@ -0,0 +1,48 @@
+export class UrlUtils {
+
+  /**
+   * Private
```

Comment:
It helps when the file grows to quickly find the private vs. public, especially when adding a method.
But if you don't like it, I'll remove it.